contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_edit_folder.handlebars

```diff
@@ -0,0 +1,18 @@
+<div id="clipboard" class="pull-right">
```

Comment:
There seems to be some redundancy here between the 'Add Folder' and the 'Edit Folder' template.

It would be better to have them unified in a single template, and then parameterize the title if needed.

You can then simply prefill the template when editing.